examples/04_scalability.py

```diff
@@ -197,3 +200,55 @@ def wrapped_func(*args, **kwargs):
 [t.set(size=16) for t in ax2.get_yticklabels()]
 plt.tight_layout()
 
+##################################################################################
+# SimilarityEncoder with default options and 32 bits float dtype
+# ---------------------------------------------------------------
+#
+# As done above we build our vectorizer using the ColumnTransformer
+# to combine one-hot encoding and similarity encoding.
+# We set the dtype parameters for both encoders to np.float32
+# and cast the column 'year' to np.float32
+# (to prevent the ColumnTransformer from force casting everything to float64).
+
+sim_enc = SimilarityEncoder(similarity='ngram', dtype=np.float32, handle_unknown='ignore')
+
+y = df['Violation Type']
+df['Year'] = df['Year'].astype(np.float32)
+
+transformers = [('one_hot', OneHotEncoder(sparse=False, dtype=np.float32, handle_unknown='ignore'),
+                 ['Alcohol',
+                  'Arrest Type',
+                  'Belts',
+                  'Commercial License',
+                  'Commercial Vehicle',
+                  'Fatal',
+                  'Gender',
+                  'HAZMAT',
+                  'Property Damage',
+                  'Race',
+                  'Work Zone']),
+                ('pass', 'passthrough', ['Year']),
+                ]
+
+column_trans = ColumnTransformer(
+    # adding the dirty column
+    transformers=transformers + [('sim_enc', sim_enc, ['Description'])],
+    remainder='drop')
+
+t0 = time()
+X = column_trans.fit_transform(df)
+t1 = time()
+print('Type of vectorized data:', X.dtype)
```

Comment:
Can you do the print in a different "cell". What I call a cell is a comment that starts with a line of "#": it is displayed specially by sphinx-gallery, which renders the example.

Also, can you do it after printing the time to vectorize.